packages/next/server/font-utils.ts

```diff
@@ -77,3 +78,58 @@ export function getFontDefinitionFromManifest(
     })?.content || ''
   )
 }
+
+function parseGoogleFontName(css: string): Array<string> {
+  const regex = /font-family: ([^;]*)/g
+  const matches = css.matchAll(regex)
+  const fontNames = new Set<string>()
+
+  for (let font of matches) {
+    const fontFamily = font[1].replace(/^['"]|['"]$/g, '')
+    fontNames.add(fontFamily)
+  }
+
+  return [...fontNames]
+}
+
+function calculateOverrideCSS(font: string, fontMetrics: any) {
+  const fontName = font.toLowerCase().trim().replace(/ /g, '-')
+  const fontKey = font.toLowerCase().trim().replace(/ /g, '')
+  const { category, ascentOverride, descentOverride, lineGapOverride } =
+    fontMetrics[fontKey]
+  const fallbackFont = category === 'serif' ? 'Times New Roman' : 'Arial'
```

Comment:
Seems like these fallbacks should be constants so they are easy to find and change, e.g. `DEFAULT_SERIF_FONT` and `DEFAULT_SANS_SERIF_FONT`

Reply:
Good point!